Services/Database/classes/QueryUtils/class.ilMySQLQueryUtils.php

```diff
@@ -288,4 +288,21 @@ public function createDatabase($a_name, $a_charset = "utf8", $a_collation = "")
 
 		return $sql;
 	}
+
+
+	/**
+	 * 
+	 * @param type $a_name
```

Comment:
Please use parameter declarations such as `string` or `int`, there is no `type`. And please rename `$a_name` to `$field_name` or something like that to make clear what name exactly is meant. Additionally `$a_order` doesn't seem to be declared in the PHPDoc.

you can write all the PHPDoc on the interface and just put a 
```
/**
 * @inheritdoc
 */
```
above the implementations.